content/rest-api/filter.md

```diff
@@ -205,6 +205,25 @@ To get all the variant products of the sub product model with the code `tshirt_a
 /api/rest/v1/products?search={"parent":[{"operator":"=","value":"tshirt_armor_blue"}]}
 ```
 
+### On their quality score
+::: availability versions=6.0,Serenity editions=CE,EE
```

Comment:
do we have to add the 6.0 information now ?

Reply:
I'm not sure either, but I don't know how to mention that the filter will be available on CE (serenity is only EE)
In any case it's not the first. The mention "6.0" is also used for the asset families: https://github.com/akeneo/pim-api-docs/blob/master/content/php-client/resources/asset-manager/asset-families.md